functions/screenshot.js

```diff
@@ -18,6 +18,36 @@
  * @param args.page - object - Puppeteer's page object (from await browser.newPage)
  * @param args.context - object - An object of parameters that the function is called with. See src/schemas.ts
  */
+const scrollThroughPage = async (page) => {
+  // Scroll to page end to trigger lazy loading elements
+  await page.evaluate(() => {
+    const scrollInterval = 100;
+    const scrollStep = Math.floor(window.innerHeight / 2);
+    const bottomThreshold = 400;
```

Comment:
Could probably be derived from https://pptr.dev/#?product=Puppeteer&version=v11.0.0&show=api-pageviewport ?

```js
const { height: bottomThreshold } = await page.viewport();
```

Reply:
Implemented